runtime/compiler/control/J9Options.cpp

```diff
@@ -2653,6 +2653,7 @@ J9::Options::fePreProcess(void * base)
    if ((omrsysinfo_get_memory_info(&memInfo) == 0) && (0 == memInfo.totalSwap))
       {
       self()->setOption(TR_EnableSelfTuningScratchMemoryUsageBeforeCompile);
+      compInfo->setIsSwapMemoryDisabled();
```

Comment:
If possible, I would move  the logic for determining swap availability in the constructor of CompInfo. Otherwise, there is a window of time where the new field `_isSwapMemoryDisabled` may have the incorrect value.

Reply:
Done.